ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +651,108 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_ocs(self):
+        """
+        The function Uninstalls the OCS operator from a openshift
+        cluster and removes all its settings and dependencies
+
+        """
+        ocp_obj = ocp.OCP()
+        provisioners = constants.OCS_PROVISIONERS
+
+        # List the storage classes
+        sc_list = get_all_storageclass()
+        for storage_class in sc_list:
+            if storage_class.get('provisioner') in provisioners:
+                sc_list.remove(storage_class)
+
+        # Query for PVCs and OBCs that are using the storage class provisioners listed in the previous step.
+        pvc_list = get_all_pvcs(namespace='all-namespaces')
+        pvcs_to_delete = []
+        for pvc in pvc_list:
+            for storage_class in sc_list:
+                if pvc.backed_sc == storage_class:
+                    pvcs_to_delete.append(pvc)
+                    break
+
+        # Removing monitoring stack from OpenShift Container Storage
```

Comment:
I would suggest to write it as a separate function something like remove_monitoring_from_stack() so that other test cases installing monitoring can use it.

Reply:
+1

Reply:
agree, where to put this functions? in this deployment file or in some other place?